docs/deprecations.md

```diff
@@ -48,6 +48,14 @@ Adapt to this behavior now by adding `useDotenv: true` to service configuration.
 
 Note that env vars are handled differently than with [serverless-dotenv-plugin](https://github.com/colynb/serverless-dotenv-plugin), check [documentation](/framework/docs/environment-variables/) for more info.
 
+<a name="ERROR_ON_UNRESOLVED_REFERENCES"><div>&nbsp;</div></a>
```

Comment:
New deprecations should be listed at the top.

Also let's maybe name code `VARIABLES_ERROR_ON_UNRESOLVED` (so code indicates it's about variables resolution